docs/index.rst

```diff
@@ -24,7 +24,8 @@ More Info
 ---------
 
 This website covers the project information for IRC Hooky such as installation,
-development, and so forth.
+development, and so forth. If you would like to kick the tires without setting
+this up yourself, have a read through the :doc:`demo <demo>`.
```

Comment:
`up yourself, it has already been set up for you to try. Have a read through....`